skiplang/prelude/src/stdlib/debug/Debug.sk

```diff
@@ -55,7 +55,7 @@ private fun debugImpl<T>(x: T, print: String -> void): void {
  *
  * For your class and return one of the values here, as appropriate.
  */
-base class .Inspect uses Hashable {
+base class .Inspect uses Hashable, Equality {
```

Comment:
Why is this needed?

Reply:
I'm investigating on that.

Reply:
Did you figure out the problem? If not, remember to make an issue summing up your investigation.

Reply:
No, I don't. It's on my plan.